sdk/storage/Azure.Storage.Blobs/src/Models/BlobDownloadOptions.cs

```diff
@@ -11,11 +11,27 @@ namespace Azure.Storage.Blobs.Models
     /// </summary>
     public class BlobDownloadOptions
     {
+        private HttpRange _range;
+        private bool _isRangeSet;
```

Comment:
make this internal instead. then below we can get rid of the `public bool IsRangeSet => _isRangeSet;` and then you will be able to access `_isRangeSet` from BlobBaseClient and PartitionedDownloader